packages/dds/merge-tree/src/test/testClient.ts

```diff
@@ -206,7 +211,29 @@ export class TestClient extends Client {
 		overwrite?: boolean;
 		opArgs: IMergeTreeDeltaOpArgs;
 	}): void {
-		this.mergeTree.obliterateRange(start, end, refSeq, clientId, seq, overwrite, opArgs);
+		const { startPos, startSide, endPos, endSide } = endpointPosAndSide(start, end);
+
+		assert(
+			startPos !== undefined &&
+				endPos !== undefined &&
+				startSide !== undefined &&
+				endSide !== undefined &&
+				startPos !== "end" &&
+				endPos !== "start",
+			"start and end cannot be undefined because they were not passed in as undefined",
+		);
+		const numericalStart = startPos === "start" ? 0 : startPos;
+		const numericalEnd = endPos === "end" ? this.getLength() - 1 : endPos;
+
+		this.mergeTree.obliterateRange(
```

Comment:
why call merge tree, most the commends just reuse the client. i think this can actually just be removed

Reply:
as in this function should call client.obliterateRange or the whole function should be removed in favor of the obliterate on client?

Reply:
what do we do for things like remove, this should be similar